### PR TITLE
ONBOARDING: Working realm functionality

### PR DESCRIPTION
**Pull request by priettt**
If the realm database is empty, characters are loaded from the server. Otherwise, they are directly loaded from the database itself. To load the characters into the database, the user needs to press the refresh button.

### CODE REVIEW

File: app/src/main/java/com/puzzlebench/cleanMarvelKotlin/presentation/mvp/CharacterPresenter.kt [hunk 1]
```diff
@@ -21,87 +21,84 @@ const val COMIC_URL_TAG = "comiclink"
...
-        getCharacter()
-        refresh()
+        retrieveFromDatabase()
+        listenToRefresh()
+        listenToDetails()
     }
 
-    private fun refresh() {
+    private fun listenToRefresh() {
         RxBus.subscribe(view.activity, object : OnRefreshPressedBusObserver() {
```
Comment (fanky10) [inline app/src/main/java/com/puzzlebench/cleanMarvelKotlin/presentation/mvp/CharacterPresenter.kt]: now that you know RXAndroid you can move away from RxBus

Comment (priettt) [reply app/src/main/java/com/puzzlebench/cleanMarvelKotlin/presentation/mvp/CharacterPresenter.kt]: done!

**Review by SartiMau — APPROVED** [review]

**Review by SartiMau — CHANGES_REQUESTED** [review]
Please put a screenshot  showing that all the tests passed

**Comment by priettt** [comment]
screen shot 2018-12-21 at 11 44 09

**Review by SartiMau — APPROVED** [review]

**Review by fanky10 — APPROVED** [review]
LG2M

**Pull request merged**